skills/delegation/references/implementer-prompt.md

```
@@ -2,6 +2,10 @@
 
 Use this template when dispatching tasks via the Task tool.
 
+### Quality Hints Integration
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)